EVA test_duplex failing on Mac

Reported by @devos50  during [PR tests](#6191 ) on Mac

```python
self = <test_eva_protocol.TestEVA testMethod=test_duplex>

    async def test_duplex(self):
        count = 100
        block_size = 10
    
        self.overlay(0).eva_protocol.block_size = block_size
        self.overlay(1).eva_protocol.block_size = block_size
    
        data0 = os.urandom(1), os.urandom(block_size * count), random.randrange(0, 256)
        data1 = os.urandom(1), os.urandom(block_size * count), random.randrange(0, 256)
    
        self.overlay(0).eva_send_binary(self.peer(1), *data0)
        self.overlay(1).eva_send_binary(self.peer(0), *data1)
    
        await drain_loop(asyncio.get_event_loop())
    
>       assert self.overlay(0).most_recent_received_data == data1
E       AssertionError: assert None == (b'N', b'\nN\...xcbw\xa8', 33)
E         +None
```

--- a/src/tribler-core/tribler_core/modules/remote_query_community/tests/test_eva_protocol.py
+++ b/src/tribler-core/tribler_core/modules/remote_query_community/tests/test_eva_protocol.py
@@ -236,6 +236,9 @@ class TestEVA(TestBase):
         count = 100
         block_size = 10
 
+        self.overlay(0).eva_protocol.terminate_by_timeout_enabled = False
+        self.overlay(1).eva_protocol.terminate_by_timeout_enabled = False
+
         self.overlay(0).eva_protocol.block_size = block_size
         self.overlay(1).eva_protocol.block_size = block_size
 

